lib/view/back_appbar.dart

```diff
@@ -32,7 +32,7 @@ AppBar getBackAppBar(
   final secondaryColor = isWhite ? AppColor.white : AppColor.primaryBlack;
   return AppBar(
     systemOverlayStyle: SystemUiOverlayStyle(
-        statusBarColor: statusBarColor ?? secondaryColor,
+        statusBarColor: backgroundColor ?? statusBarColor ?? secondaryColor,
```

Comment:
the `statusBarColor` should have more priority than `backgroundColor`

Reply:
yeah, you're right

Reply:
forgot that we also pass statusBarColor